Tribler/Core/Modules/restapi/search_endpoint.py

```diff
@@ -116,6 +116,8 @@ def search_db():
 
         def on_search_results(search_results_tuple):
             search_results, total = search_results_tuple
+            if not search_results:
```

Comment:
There's no JSON response here?

Reply:
Hmm... Maybe you're right, and there should be a zero-results response instead. I used this thing as a temporary workaround for a problem.